terraform/cloudwatch_dashboard.template.json.py

```diff
@@ -717,6 +831,19 @@ def dashboard_body(name: str):
                                 'region': config.region
                             }
                         ]
+                    ]
+                    if name == 'indexer' else
+                    [
+                        [
+                            'AWS/Lambda',
+                            'Errors',
+                            'FunctionName',
+                            config.indexer_function_name('mirror'),
+                            {
+                                'label': 'mirror',
+                                'region': config.region
+                            }
+                        ]
```

Comment:
Nit: consider refactoring this entire section to use a comprehension loop.

Reply:
Alternatively, the indexer widgets could be made less repetitive using the `...` syntax, if I'm understanding the structure of the other widgets correctly.